File: Course_01/C Language/EstruturaCondicional/operadora.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    double min, total;

    printf("Digite a quantidade de minutos: ");
    scanf("%lf", &min);

    if (min<=100) {
        total=50;
    }
    else {
        total=50+((min-100)*2);
    }

    printf("Valor a pagar: %.2lf", total);

    return 0;
}

File: Course_01/Java Language/Estrutura condicional/operadora.java
package condicional;

import java.util.Locale;
import java.util.Scanner;

public class operadora {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int tempo; 
		double total;
		
		System.out.print("Digite a quantidade de minutos: ");
		tempo = sc.nextInt();
		
		if (tempo > 100) {
			total = (double) 50 + (tempo - 100) * 2;
		}
		else {
			total = (double) 50;
		}
		
		System.out.println("Valor a pagar: R$ " + String.format("%.2f", total));
		
		sc.close();
	}

}
